packages/web/src/components/RegistrationModal/HintTooltip.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Box, Tooltip } from '@chakra-ui/react';
+import { colors } from '../../theme/colors';
+
+export const Hint: React.FC<{ children: string }> = ({ children }) => (
```

Comment:
File name is inconsistent with the export; please make sure the two are identical but I don't have a preference as to which one

Also the type on this `children` prop is incorrect; it CAN be a string but is not guaranteed to be one. It also has a separate type than the type of `children` everywhere else.
```suggestion
type HintProps = PropsWithChildren;

export const Hint: React.FC<HintProps> = ({ children }) => (
```